include/dlaf/factorization/qr/t_factor_impl.h

```diff
@@ -72,9 +71,9 @@ struct Helpers<Backend::MC, Device::CPU, T> {
                          });
   }
 
-  static matrix::Tile<T, Device::CPU> gemvLoop(const matrix::Tile<const T, Device::CPU>& tile_v,
-                                               const matrix::Tile<const T, Device::CPU>& taus,
-                                               matrix::Tile<T, Device::CPU> tile_t) noexcept {
+  static matrix::Tile<T, Device::CPU> loop_GEMV(const matrix::Tile<const T, Device::CPU>& tile_v,
```

Comment:
Just a style question:
```suggestion
  static matrix::Tile<T, Device::CPU> loop_gemv(const matrix::Tile<const T, Device::CPU>& tile_v,
```
?

Reply:
see 05dc48d